Source/WebCore/dom/TrustedTypePolicyFactory.idl

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 Igalia S.L. All rights reserved.
```

Comment:
Are we sure we're not copying any Blink files here?
If so, copyright notice does need to mention Chromium authors.

Reply:
I'm fairly confident these files are all either IDL copied from the spec or written from scratch.